.github/workflows/ruby.yml

```diff
@@ -21,9 +21,12 @@ jobs:
           - 3.0.6
           - 3.1.4
         mongo-image:
-          - mongo:4.2
           - mongo:4.4
-          - mongo:5.0
+        include:
+          - { "ruby-version": 3.1.4, "mongo-image": "mongo:4.2" }
+          - { "ruby-version": 3.1.4, "mongo-image": "mongo:5.0" }
+          - { "ruby-version": 3.1.4, "mongo-image": "mongo:6.0" }
+          - { "ruby-version": 3.1.4, "mongo-image": "mongo:7.0" }
```

Comment:
It takes too long time to complete conditions for all Ruby versions by all MongoDB versions. So I changed the configuration to run all MongoDB versions only with Ruby 3.1.